src/features/rewards/walletWrapper/style.ts

```diff
@@ -2,13 +2,16 @@
  * License. v. 2.0. If a copy of the MPL was not distributed with this file.
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { Notification, NotificationType } from './'
 import styled from 'styled-components'
 
 interface StyledProps {
   connected?: boolean,
   contentPadding?: boolean,
   compact?: boolean,
-  background?: string
+  background?: string,
+  type?: NotificationType,
```

Comment:
can we have just one type for notification

```
notification?: Notification | undefined
``` 

type is already part of `Notification` object